### PR TITLE
Add Erwtensoep (Dutch pea soup) recipe

### PR DESCRIPTION
**Pull request by rijkvp**

### CODE REVIEW

File: src/erwtensoep.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- 2 ribs celery
+- 2 to 3 carrots (peeled and sliced)
+- 1 large potato (peeled and cubed)
+- 1 small onion (chopped)
+- 1 small leek (sliced)
+- 1 cup celeriac (cubed)
+- 1 pound sausage (chopped rookworst, smoked sausage)
+- Salt and pepper
+
+## Directions
```
Comment (hallzy) [inline src/erwtensoep.md:28]:
```suggestion
- 1 pound sausage (chopped rookworst, smoked sausage)

## Directions
```

Remove salt/pepper from ingredients list. They can stay in the directions

File: src/erwtensoep.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+1. In a large soup pot, bring water, split peas, pork belly or bacon, pork chop, and bouillon cube to a boil.
+2. Reduce the heat to a simmer, cover and let cook for 45 minutes, stirring occasionally and skimming off any foam that rises to the top. 
+3. Remove the pork chop, debone, and thinly slice the meat. Set aside.
+4. Add the celery, carrots, potato, onion, leek, and celeriac to the soup.
+5. Return to the boil, reduce the heat to a simmer and let cook, uncovered, for another 30 minutes
```
Comment (hallzy) [inline src/erwtensoep.md:34]:
```suggestion
5. Return to a boil, reduce the heat to a simmer and let cook uncovered for another 30 minutes.
```

**Review by hallzy — CHANGES_REQUESTED** [review]
Do you have an image that can be added?

File: src/erwtensoep.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- 1 1/2 cups / 300 grams peas (dried green split)
+- 3 1/2 ounces / 100 grams pork belly (Dutch speklapjes or thick-cut bacon)
+- 1 pork chop
+- 1 bouillon cube (vegetable, pork or chicken)
+- 2 ribs celery
+- 2 to 3 carrots (peeled and sliced)
+- 1 large potato (peeled and cubed)
+- 1 small onion (chopped)
+- 1 small leek (sliced)
+- 1 cup celeriac (cubed)
```
Comment (printplaatreparatie) [inline src/erwtensoep.md:24]:
```suggestion
- 1 celeriac (cubed)
```

**Pull request merged**